src/workerd/api/web-socket.h

```diff
@@ -425,6 +425,9 @@ class WebSocket: public EventTarget {
   // `close()`, thereby preventing calls to `send()` even after we wake from hibernation.
   bool closedOutgoingForHib = false;
 
+  // Maximum allowed size for WebSocket messages
+  size_t maxMessageSize = 1u << 20;
```

Comment:
Use the constant `kj::WebSocket::SUGGESTED_MAX_MESSAGE_SIZE`.

Reply:
https://github.com/cloudflare/workerd/pull/2164/commits/bfbfda1c06ba3934725d1312f2e0a0fdd1546472